File: src/test/java/com/lee/PersonTest.java
package com.lee;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;


public class PersonTest {

    @Test
    public void bmiTester(){
        Person person = new Person("Lee",66.5f,1.7f);
        float bmi = 66.5f/(1.7f*1.7f);
        Assertions.assertEquals(bmi,person.bmi());

    }


}

File: src/test/kotlin/com/lee/PersonTest.kt
package com.lee.kotlin

import com.lee.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test


class PersonTest{
    val student= Student("jack",77,50);


    @Test
    fun bmiTest(){
        Assertions.assertEquals(77,student.highest())
    }

    @Test
    fun getAverageTest(){
        Assertions.assertEquals((77+50)/2,student.getAverage())
    }

    @Test
    fun passOrFailedTest(){
        Assertions.assertEquals("PASS",student.passOrFailed())
    }

    @Test
    fun gardingTest(){
        Assertions.assertEquals('D',student.garding())
    }
}